db/migrate/20161121231919_follow_topics_on_create.rb

```diff
@@ -0,0 +1,6 @@
+class FollowTopicsOnCreate < ActiveRecord::Migration
```

Comment:
Let's move this to a new migration file in `db/upgrade_migrations/#######_upgrade_v0_9_to_v0_10.rb`, and update the [create_thredded](https://github.com/thredded/thredded/blob/master/db/migrate/20160329231848_create_thredded.rb) migration.